panel/src/components/Forms/Field/StructureField.vue

```diff
@@ -563,6 +564,13 @@ export default {
 			} else {
 				return true;
 			}
+		},
+		/**
+		 * When the field's value changes
+		 * @param {array} values
+		 */
+		onFormInput(e) {
+			this.$emit("onFormInput", e);
```

Comment:
To be consistent with other events, would be good if this only emits `formInput` without `on`.